api/controller/readcsv_controller.go

```diff
@@ -0,0 +1,57 @@
+package controller
```

Comment:
Also if you're renaming this file to reflect handling contestants, it could be named `contestants.go`